.travis.yml

```diff
@@ -32,14 +36,7 @@ addons:
         - libtool-bin
 
 install:
-    - ARCH=`uname -p`
-    - echo $ARCH
-    - JDK_X64="https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u242-b08/OpenJDK8U-jdk_x64_linux_hotspot_8u242b08.tar.gz"
-    - JDK_ARM64="https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u232-b09/OpenJDK8U-jdk_aarch64_linux_hotspot_8u232b09.tar.gz"
-    - if test "X$ARCH" = "Xaarch64"; then JDK_URL=$JDK_ARM64; else JDK_URL=$JDK_X64; fi
-    - wget -q $JDK_URL && tar xzf OpenJDK*.tar.gz
-    - mv jdk8* jdk
-    - export JAVA_HOME=`pwd`/jdk
```

Comment:
The above was here for two reasons:
1) because TravisCI is really inconsistent in what is available in the different types of VMs
1.1) JAVA_HOME is exported for AMD64, but not for ARM64
1.2) `mvn` is available for AMD64, but not for ARM
1.3.) most important - to keep the version of the used JDK the same for all environments. The one provided by Travis for ARM64 was older than the one for AMD64

For the above reasons I'd prefer to keep the control

Reply:
I had shifted to use travis jdk to make things cleaner. Also since the version was exactly same i thought the steps were redundant. I will revert since you want keep it the way it is.